src/util/useCheckCanDelete.ts

```diff
@@ -0,0 +1,17 @@
+import useOktaTokens from "../hooks/useOktaTokens";
+
+const useCheckUserCanDelete = (
+  createdBy: string,
```

Comment:
I think this could be user or userName.g.
```suggestion
  user: string,
```

Reply:
thank you, I'll make the changes